src/ui/Button.tsx

```diff
@@ -1,8 +1,10 @@
 import { forwardRef, ReactNode } from "react";
 import { useNavigate } from "react-router-dom";
 import styled from "styled-components";
+import iconDownload from "./icon-download.svg";
+import iconGo from "./icon-go.svg";
 
-type ButtonType = "primary" | "primary-blue" | "secondary" | "tertiary";
+type ButtonType = "primary" | "primary-blue" | "secondary" | "secondary-download" | "tertiary";
```

Comment:
Why is it that we need a different "secondary-download" type? 

Looking at the design for Resources page, those download buttons to me look like the exact style for the button called "secondary" in the design system anyway?

Reply:
"secondary" doesn't include the download icon; "secondary-download" does.

I added the "secondary-download" button type in response to a comment above: "We don't know if all future secondary buttons will all get a download icon."

If ever we need a "secondary" button that shouldn't have a download icon, we can use "secondary."
Meanwhile, the name "secondary-download" signals that it should only be used for buttons that let you download stuff.